### PR TITLE
added pull request template

### PR DESCRIPTION
**Pull request by salmanmkc**
Pull request template to make it easier to contribute

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Fixes
```
Comment (adiazulay) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
## Description

## Fixes
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Type of PR
+
```
Comment (adiazulay) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+## What's happening now?
+
+## What's changed?
+
+## Checklist
```
Comment (adiazulay) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
## Checklist
- [ ] Code follows this projects style guide
- [ ] I reviewed my code
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+- [ ] Tested in deployment environment
+- [ ] Tested in an alternate environment (please update here: )
+- [ ] Major feature added
```
Comment (adiazulay) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Required documentation has been updated
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+- [ ] Bux fix
```
Comment (adiazulay) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+## Fixes
+
+## Type of PR
+
+## What's happening now?
+
+## What's changed?
```
Comment (adiazulay) [inline .github/PULL_REQUEST_TEMPLATE.md]: Do you think we can capture" What's happening now" and "What's changed" by adding a description field at the top instead?

**Review by adiazulay — CHANGES_REQUESTED** [review]
This looks great, thanks for contributing! I requested a few changes to make the template more succinct.

**Review by yaynoel — APPROVED** [review]
Thanks for the contribution! +1 to Adi's recommendation, I think the two questions, What's happening now?, What's changed? could be combined into Description.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (salmanmkc) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thanks for those suggestions, they are great! Anything else to add? 😊

**Review by adiazulay — APPROVED** [review]

**Pull request merged**